projects/batfish-client/src/main/java/org/batfish/client/Client.java

```diff
@@ -792,6 +799,10 @@ private boolean answerType(
 
    private boolean cat(String[] words)
          throws IOException, FileNotFoundException {
+      if (words.length == 1) {
+         _logger.errorf(words[0] + " requires additional arguments\n");
```

Comment:
Every one of these errors should be descriptive.

E.g., it should not be "cat requires additional parameters". It should be "usage: cat <filename>" or something like that.